README.md

```diff
@@ -72,6 +72,80 @@ resource "aws_launch_configuration" "multipart" {
 }
 ```
 
+### Example with using the iam_policy_document and aws_iam_role:
+
+```hcl
+locals {
+  application {
+    name      = "cloudwatch_agent"
+    stage     = "dev"
+    namespace = "eg"
+  }
+}
+
+module "label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
+  environment = "${local.application["stage"]}"
+  name        = "${local.application["name"]}"
+  namespace   = "${local.application["namespace"]}"
+}
+
+module "cloudwatch_agent" {
+  source = "git::https://github.com/cloudposse/terraform-aws-cloudwatch-agent?ref=master"
+
+  name      = "${module.label.name}"
+  stage     = "${module.label.environment}"
+  namespace = "${module.label.namespace}"
+}
+
+resource "aws_launch_configuration" "multipart" {
+  name_prefix          = "${module.label.name}"
+  image_id             = "${data.aws_ami.ecs-optimized.id}"
```

Comment:
should we add `data.aws_ami` to the example, or provide some real `image_id`?

Reply:
:+1: